src/vs/workbench/contrib/scm/browser/media/scm.css

```diff
@@ -194,6 +194,19 @@
 .scm-view .scm-input {
 	height: 100%;
 	padding-left: 11px;
+	border-radius: 2px;
```

Comment:
@lszomoru I'm having a heck of a time applying padding and border radius rules to this SCM input. It seems the "rest" and active states are using different styles, some of which seem cryptically inlined from `.monaco-editor`. 

I'd like to avoid all of these `!important` rules if possible—any ideas on how to do this cleanly?

Reply:
See entire file diff for examples. Attached the comment to the wrong line.